functions/package.json

```diff
@@ -20,14 +21,17 @@
     "meilisearch": "0.19.0"
   },
   "devDependencies": {
+    "@babel/preset-typescript": "^7.15.0",
+    "@types/jest": "^27.0.1",
     "@typescript-eslint/eslint-plugin": "^3.9.1",
     "@typescript-eslint/parser": "^3.8.0",
     "eslint": "^7.6.0",
     "eslint-config-google": "^0.14.0",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-import": "^2.22.0",
     "eslint-plugin-prettier": "^3.4.0",
-    "firebase-functions-test": "^0.2.0",
+    "firebase-functions-test": "^0.2.3",
+    "jest": "^27.0.6",
```

Comment:
I don't see any tests inside the project, will you implement them later ?

Reply:
Yes, indeed.